ert_gui/model/snapshot.py

```diff
@@ -238,6 +238,24 @@ def _add_partial_snapshot(self, partial: PartialSnapshot, iter_: int):
                                     attr, job.get(ids.DATA).get(attr)
                                 )
 
+                    if jobs_changed:
+                        job_top_left = self.index(min(jobs_changed), 0, step_index)
+                        job_bottom_right = self.index(
+                            max(jobs_changed),
+                            self.columnCount(step_index) - 1,
+                            step_index,
+                        )
+                        stack.callback(
+                            self.dataChanged.emit, job_top_left, job_bottom_right
+                        )
+
+            if reals_changed:
+                real_top_left = self.index(min(reals_changed), 0, iter_index)
+                real_bottom_right = self.index(
+                    max(reals_changed), self.columnCount(iter_index) - 1, iter_index
+                )
+                stack.callback(self.dataChanged.emit, real_top_left, real_bottom_right)
```

Comment:
There is a tradeoff being made here. If say only two realizations are changed, the first and the last one in a large ensemble, this `emit` will target every realization with this PR instead of only the two in question.

Reply:
The tradeoff is ok. Realizations are dominantly consecutive, and the cost of updating redundant indices in QT is probably small.